### DISCUSSION TITLE
🚀 Structure - option to create pages that link to already created pages.

### DISCUSSION THREAD

**Discussion by DesignWorkshop** [ExpressionEngine/ExpressionEngine#4595] [Feature Requests]
### Suggest an idea for ExpressionEngine

In the attached screenshot I would like to create a child page under 'What we fund' called 'Research trials' that links to the 'Research trials' page under 'Our work'.
<img width="853" alt="Screenshot 2024-11-14 at 15 17 50" src="https://github.com/user-attachments/assets/6c8ad1cf-3688-4623-a097-dcc58490e59f">

**Comment by intoeetive** [comment]
You can create custom field of Structure type and use it to link to a page. The output of the field will be page URI
